src/main/java/file/writer/FileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package file.writer;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String report, String fileName) {
+        try (
+                PrintWriter reportWriter = new PrintWriter(fileName)) {
+            reportWriter.print(report);
+        } catch (IOException e) {
+            throw new RuntimeException();
```

Comment:
Catch block throws a general RuntimeException without an informative message or the cause. According to the checklist, you should rethrow a RuntimeException with an informative message and exception object.